fairlearn/preprocessing/_correlation_remover.py

```diff
@@ -61,7 +63,7 @@ class CorrelationRemover(BaseEstimator, TransformerMixin):
 
     """
 
-    def __init__(self, *, sensitive_feature_ids=None, alpha=1):
+    def __init__(self, *, sensitive_feature_ids: Iterable = (), alpha: float = 1):
```

Comment:
At first I tried to leave the `None` unchanged and mutate it to an empty list within the `__init__`, but the `sklearn` compatibility check didn't like that :sweat_smile: 

So I put an empty `tuple` (for immutability) and allowed any `Iterable` instead of restricting the argument to be a list (I double checked that the code can indeed handle any `Iterable`).